native/src/main/java/io/ballerina/stdlib/serdes/protobuf/ProtobufMessageFieldProperties.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.stdlib.serdes.protobuf;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.google.protobuf.DescriptorProtos.FieldDescriptorProto.Label;
+import static com.google.protobuf.DescriptorProtos.FieldDescriptorProto.Type;
+
+/**
+ * Provides String to protobuf field properties mapping.
+ * Properties include protobuf field types and labels.
+ */
+public class ProtobufMessageFieldProperties {
```

Comment:
Since everything under this class is static, let's add a private constructor to this. The code-coverage warning can also be avoided by this.

Reply:
Added a private constructor